test/test.js

```diff
@@ -74,4 +74,23 @@ describe('markdown-it-mfr', function () {
     id = getMfrId(renderedHtml);
     assert.equal(renderedHtml, '<p><div id="' + id + '" class="mfr mfr-file"></div><script>$(document).ready(function () {new mfr.Render("' + id + '", "https://mfr.osf.io/render?url=https://osf.io/xxxxx/?action=download%26mode=render");    }); </script></p>\n');
   });
+
+  it('make sure normal iframe generates properly with link to staging', function () {
```

Comment:
You don't need to change it for this, and I should have caught it the first go through, but in the future, if you're doing BDD for testing, it should read like a sentence. "It make sure normal iframe generates…" is a could description of the behavior of your app. Instead, something like, "It generates iframe properly when using staging mfr link" would be the way to go on that.